server/game/gamesteps/challenge/challengeflow.js

```diff
@@ -61,7 +61,17 @@ class ChallengeFlow extends BaseStep {
     }
 
     chooseAttackers(player, attackers) {
-        this.challenge.addAttackers(attackers);
+        this.attackersToKneel = [];
+        this.challenge.addAttackers(attackers, false);
```

Comment:
It may be worth looking into whether we need the `kneel` second parameter now when adding attackers / defenders. My guess it's nearly always `false` now and the places it isn't the character may already be knelt as a cost of the ability.